snapm/snapm_exists_error.py

```
@@ -0,0 +1,37 @@
+
+        raise SnapmExistsError("my-snapset")
+        # SnapmExistsError: Snapshot set 'my-snapset' already exists
+
+    Raising with a custom message::
+
+        raise SnapmExistsError("my-snapset", "my-snapset is taken; choose another name")
+    """
+
+    _default_template = "Snapshot set {name!r} already exists"
+
+    def __init__(self, name: str, msg: str = None) -> None:
```

Ruff (0.16.0)

[warning] PEP 484 prohibits implicit `Optional`

Convert to `Optional[T]`

(RUF013)